scripts/rebuild-all

```diff
@@ -1,24 +1,20 @@
 #!/bin/bash
 
-echo ">> Moving into deltachat-core"
-pushd deltachat-core
+set -e
 
-echo ">> Recreating builddir/"
-sudo rm buildir/ -rf && mkdir builddir
+echo ">> Recreating deltachat-core/builddir/"
+rm deltachat-core/builddir -rf && mkdir -p deltachat-core/builddir
 
-echo ">> Moving into builddir"
-pushd builddir
+echo ">> Moving into deltachat-core/builddir"
+pushd deltachat-core/builddir
 
+echo ">> meson"
+meson --default-library=static
 echo ">> ninja"
-ninja
-echo ">> sudo ninja install"
-sudo ninja install
-echo ">> sudo ldconfig"
-sudo ldconfig
+ninja -v
```

Comment:
This causes ninja to spam a lot. We should remove this later, but useful now to see what's going on when the c code is being built.